packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2756,8 +2786,23 @@ abstract class Element implements BuildContext {
   Set<InheritedElement> _dependencies;
   bool _hadUnsatisfiedDependencies = false;
 
+  void _debugCheckStateIsActiveForAncestorLoopkup() {
```

Comment:
(you can also just have this return true, then the asserts can be simpler. no biggie.)